comp/dogstatsd/server/server.go

```diff
@@ -29,7 +29,7 @@ import (
 	replay "github.com/DataDog/datadog-agent/comp/dogstatsd/replay/def"
 	serverdebug "github.com/DataDog/datadog-agent/comp/dogstatsd/serverDebug"
 	"github.com/DataDog/datadog-agent/pkg/aggregator"
-	"github.com/DataDog/datadog-agent/pkg/config"
+	"github.com/DataDog/datadog-agent/pkg/config/model"
```

Comment:
Why change this to model?

Reply:
So we stop using the aliases but the real package in the config space. This reduce dependencies and highligh the real dependency graph.